File: ACADEMIC CODES/Array/javascript/array_search.js
// JavaScript Implementation
// Array Search
// Searching An Element at a specific location


// Search Function
function findElement( arr, n, key)
{
    let i;
    for (i = 0; i < n; i++)
        if (arr[i] == key)
            return i;
    return -1;
}

let array1 = [1, 3, 5, 6, 4];

let n = array1.length;

let search_item = 6;
let position = findElement(array1, n, search_item);


if (position == - 1)
    console.log("Element not found");
else
    console.log('Element Found');

File: ACADEMIC CODES/Array/c++/array_search.cpp
// C++ Implementation
// Array Search
// Searching An Element at a specific location

#include <iostream>
using namespace std;

int main() {
    int array1[] = {1,2,3,4,5,6,7,8,9,10};
    int item = 5; 
    int n = 10;
    int j = 0;
    

    while( j < n){
        if( array1[j] == item ) {
            break;
        }
		
        j = j + 1;
    }
	
    printf("Element found: %d at position %d\n", item, j+1);
}